tests/cluster_tests/RavenRunsRaven/Code/inner.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<Simulation verbosity="debug">
+<Simulation verbosity="debug" profile="jobs">
```

Comment:
I think TestInfo need to be added, and the revisions node need to be updated to reflect the python command change.

Reply:
FYI, this isn't a test file, it's a file run by a test file. When we first added this test we had a discussion about it, and decided that the "inner" of the RrR tests should not be considered the test file; rather the "outer" should. I think the philosophical idea was that the "outer" is actually the test, while the "inner" doesn't get seen by the testing harness, and it would be confusing if this data was loaded as if it were a separate test into the regression test documentation.

Reply:
sounds good to me.